src/app/events/[slug]/components/SponsorGrid.tsx

```diff
@@ -0,0 +1,31 @@
+import { clsx } from 'clsx'
+
+import { type SponsorLogoProps, SponsorLogo } from './SponsorLogo'
+import type { Sponsor } from './SponsorSection'
+
+type SponsorGridProps = {
+  sponsors: Array<Sponsor>
+  tier: string
+  gridClassName: string
+  logoImageConfig: SponsorLogoProps['logoImageConfig']
+}
+
+export function SponsorGrid({
+  sponsors,
+  tier,
+  gridClassName,
+  logoImageConfig,
+}: SponsorGridProps) {
+  return (
```

Comment:
Think we should add a condition before the return statement 
```
  if (sponsors.length === 0) {
    return null
  }
```

Otherwise it renders an empty div and add extra space between the title and grid (because of the `gap-8` that wraps the logos)

<img width="1260" alt="Screenshot 2024-09-26 at 14 34 15" src="https://github.com/user-attachments/assets/9575ea8e-2081-49e1-a561-2ac8d406cfb2">

Reply:
I;m unable to reproduce this..

![CleanShot 2024-09-26 at 15 43 24@2x](https://github.com/user-attachments/assets/7f80a895-3043-49f6-8fa8-5cf5a0651de2)

Reply:
Ah interesting. I've removed the first tier, so it only had second and third

Reply:
Here's the empty div
<img width="1260" alt="Screenshot 2024-09-26 at 15 53 20" src="https://github.com/user-attachments/assets/3b4c529b-102b-46a9-a248-90fae54fd372">